app/code/Magento/Captcha/Model/Customer/Plugin/AjaxLogin.php

```diff
@@ -16,70 +21,70 @@
 class AjaxLogin
 {
     /**
-     * @var \Magento\Captcha\Helper\Data
+     * @var CaptchaHelper
      */
-    protected $helper;
+    private $helper;
 
     /**
-     * @var \Magento\Framework\Session\SessionManagerInterface
+     * @var SessionManagerInterface
      */
-    protected $sessionManager;
+    private $sessionManager;
 
     /**
      * @var JsonFactory
      */
-    protected $resultJsonFactory;
+    private $resultJsonFactory;
 
     /**
-     * @var \Magento\Framework\Serialize\Serializer\Json
+     * @var Json
      */
-    protected $serializer;
+    private $serializer;
 
     /**
      * @var array
      */
-    protected $formIds;
+    private $formIds;
 
     /**
      * @param CaptchaHelper $helper
      * @param SessionManagerInterface $sessionManager
      * @param JsonFactory $resultJsonFactory
      * @param array $formIds
-     * @param \Magento\Framework\Serialize\Serializer\Json|null $serializer
-     * @throws \RuntimeException
+     * @param Json $serializer
      */
     public function __construct(
         CaptchaHelper $helper,
         SessionManagerInterface $sessionManager,
         JsonFactory $resultJsonFactory,
         array $formIds,
-        \Magento\Framework\Serialize\Serializer\Json $serializer = null
+        Json $serializer
```

Comment:
Could you alias that to `JsonSerializer`? It's unclear when only `Json` is used if it's `Result\Json` or `Serialize\Json` or other `Json`-type element